doc/sphinx/source/introduction.rst

```diff
@@ -50,12 +50,13 @@ these challenging objectives the ESMValTool is conceived as a framework
 which allows community contributions to be bound into a coherent
 framework.
 
-.. _contact:
+.. _Support-and-Contact:
 
-Contact
+Support
 =======
 
-See `www.esmvaltool.org <https://www.esmvaltool.org>`_ for general contact information.
+Support for ESMValTool can be found here `ESMValTool Discussions page <https://github.com/ESMValGroup/ESMValTool/discussions>`__
+where users can open an issue and a member of the [User Engangement Team] (mailto:esmvaltool_user_engagement_team@listserv.dfn.de) of ESMValTool will reply as soon as possible.
```

Comment:
A technical suggestion to display the email address:
```suggestion
where users can open an issue and a member of the 
`User Engagement Team <mailto:esmvaltool_user_engagement_team@listserv.dfn.de>`_
 of ESMValTool will reply as soon as possible.
```

Reply:
But do we want to say explicitly that people can open a GitHub Discussion and write us at the UET email adress? For example:
"Support for ESMValTool can be found here `ESMValTool Discussions page <https://github.com/ESMValGroup/ESMValTool/discussions>`__
where users can open an issue.
Additionally, the User Engagement Team can be contacted at its email address
(mailto : esmvaltool_user_engagement_team@listserv.dfn.de)"

Any opinion @ESMValGroup/userengagementteam?